magento/pwa-studio
# Add legal header character

## Description

When trying to add an upward rule based on user-agent it fails due to the `-` character.

## Related Issue





Closes #2485 .

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. add upward rule using the user-agent
```
  resolver: conditional
  when:
    - matches: request.headers.user-agent
```
2. observe message "Error: Illegal context property name found: request.headers.user-agent\nContext properties must be dot-separated strings and contain only letters, numbers, and underscores, and cannot begin with a dot." is no longer present

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
